web/web-mock/web-mock-rest/src/main/java/com/castlemock/web/mock/rest/controller/mock/AbstractRestServiceController.java

```diff
@@ -512,6 +458,10 @@ protected RestResponse mockResponse(final RestRequest restRequest,
             }
         }
 
+        if (mockResponse == null && restMethod.getForwardedEndpoint() != null) {
```

Comment:
Could be nice to check if the restMethod is configured to forward upon no matching mocked responses.